codalab/worker/worker_run_state.py

```diff
@@ -279,6 +295,12 @@ def mount_dependency(dependency, shared_file_system):
             status_messages.append(
                 'Pulling docker image: ' + (image_state.message or docker_image or "")
             )
+            # if "message" in self.docker_image_manager._downloading[docker_image]:
+
+            if image_state.message:
```

Comment:
I don't think this code is needed at all. The code above (`'Pulling docker image: ' + (image_state.message or docker_image or "")`) already sets the message. We can also remove `get_message_of_download_status` then.

Reply:
sgtm